app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Tag, Tooltip, Avatar, Dropdown } from "neetoui";
+import { Header } from "neetoui/layouts";
+
+import { USER_AVATAR_URL } from "../constants";
+import { dateTimeFormatter, relativeCreationTime } from "../utils";
+
+const { Menu, MenuItem } = Dropdown;
+const Cards = ({ item }) => (
+  <div className="m-3 w-full  border border-gray-200 p-4 shadow-sm  dark:border-gray-500 dark:bg-gray-600">
```

Comment:
There are some unnecessary/extra whitespaces in between each classname. Remove the unnecessary whitespaces.